demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -391,20 +394,19 @@ def execute_all_packages(self,
                             return_warning_code += pkg_status["warning_code"]
                     if pkg_status["pack_type"] not in pkgs_type:
                         pkgs_type.append(pkg_status["pack_type"])
+                logger.info('Finished all futures')
                 return return_exit_code, return_warning_code
         except KeyboardInterrupt:
-            print_warning("Stop demisto-sdk lint - Due to 'Ctrl C' signal")
-            try:
-                executor.shutdown(wait=False)
-            except Exception:
-                pass
+            msg = "Stop demisto-sdk lint - Due to 'Ctrl C' signal"
+            print_warning(msg)
+            logger.warning(msg)
+            executor.shutdown(wait=False)  # If keyboard interrupt no need to wait to clean resources
             return 1, 0
         except Exception as e:
-            print_warning(f"Stop demisto-sdk lint - Due to Exception {e}")
-            try:
-                executor.shutdown(wait=False)
-            except Exception:
-                pass
+            msg = f"Stop demisto-sdk lint - Due to Exception {e}"
+            print_warning(msg)
+            logger.warning(msg)
+            executor.shutdown(wait=True)  # wait for the resources to be cleaned. Note that `cancel_futures` not supported in python 3.8
```

Comment:
in case something will be wrong with the lint( e.g. docker connection error etc.), `shutdown(wait=True)` can cause a lot of error messages to be printed.
this will mess up the stdout.

Reply:
so? let it print the error messages. Now we are debugging in the dark.

Reply:
The result might be ~1700 duplicates of the same error message.
Once we have a failure for some reason we don’t really need to continue.
I can’t think about a case that these logs will give us any extra information.

Reply:
added the `cancel_futures` when the python version is above 3.9.
On 3.8, will manually cancel all the futures.